oan-etl/src/test/java/org/jax/oan/ontology/HpoOntologyAnnotationLoaderTest.java

```diff
@@ -126,16 +109,30 @@ void diseaseToGene() {
 	void diseaseToPhenotype() {
 		try(Session session = driver.session()) {
 			List<Node> allAnnotations = session.run("MATCH " +
-							"(n: Disease)-[:MANIFESTS]-(p: Phenotype)-[:WITH_METADATA]-(pm:PhenotypeMetadata) RETURN DISTINCT pm")
+							"(n: Disease)<-[:MANIFESTS]-(p: Phenotype)<-[:DESCRIBES {context: n.id}]-(pm: PhenotypeAnnotation) RETURN pm")
 					.list(record -> record.get("pm").asNode());
 			List<Node> filteredAnnotations = session.run("MATCH " +
-							"(n: Disease {id: 'OMIM:609153'})-[:MANIFESTS]-(p: Phenotype)-[:WITH_METADATA {context: n.id}]-(pm: PhenotypeMetadata) RETURN DISTINCT pm")
+							"(n: Disease {id: 'OMIM:609153'})<-[:MANIFESTS]-(p: Phenotype)<-[:DESCRIBES {context: n.id}]-(pm: PhenotypeAnnotation) RETURN pm")
 					.list(record -> record.get("pm").asNode());
 
 			assertEquals(5, allAnnotations.size());
 			assertEquals(3, filteredAnnotations.size());
 		}
+	}
 
+	@Test
+	void medicalActions(){
```

Comment:
I notice that you have a preference not to use the private or protected keywords. These are your friends, or would be if you just gave them a chance.